index.d.ts

```diff
@@ -1,4 +1,6 @@
 import inquirer = require("inquirer");
+import {IOptions as GlobOptions} from 'glob';
```

Comment:
You're going to want the types from `'globby'`, not `'glob'`, as that's what plop uses.  Try this instead:

```typescript
// @types/globby doesn't export types for GlobOptions, so we have to work a little bit to extract them:
// GlobOptions is the second parameter of the sync function, which can be extracted with the Parameters<T> type
import {sync as _sync} from 'globby';
type GlobOptions = Parameters<typeof _sync>[1];
```

Reply:
Oooh, okay. Gotcha, my bad. Admittedly haven't used the glob lib with TS projects much. Should have this updated by tonight

Reply:
All good, man.  This comes from my old, abandoned attempt at what you're doing.  Glad to see someone else leading the charge!